### DISCUSSION TITLE
Support for include command for static and dynamic configs

### DISCUSSION THREAD

**Discussion by brandousov** [bluenviron/mediamtx#283] [Questions]
## Which version are you using?

v0.12.2

## Enhancement request

Hi! Some of RTSP streams are updated dynamically via our custom REST API service, but some streams and base config stay unchanged. We have some risks to lose all server configuration when saving modified single cfg file. It would be nice to have base cfg file and 2 included - with static and API-modified streams (like nginx, php-fpm and others). Thank you!

PS: We also need Config API too :) #156

**Comment by aler9** [comment]
My current advice is to build the REST API to write the configuration into another, temporary, file, and then place it into the right position with the move command / function.
But at this point, i think that the config API must be implemented.

[a comment by an automated account is not shown]
